static/app/components/feedback/list/feedbackListBulkSelection.tsx

```diff
@@ -0,0 +1,76 @@
+import Button from 'sentry/components/actions/button';
+import {DropdownMenu} from 'sentry/components/dropdownMenu';
+import ErrorBoundary from 'sentry/components/errorBoundary';
+import decodeMailbox from 'sentry/components/feedback/decodeMailbox';
+import useBulkEditFeedbacks from 'sentry/components/feedback/list/useBulkEditFeedbacks';
+import type useListItemCheckboxState from 'sentry/components/feedback/list/useListItemCheckboxState';
+import {Flex} from 'sentry/components/profiling/flex';
+import {IconEllipsis} from 'sentry/icons/iconEllipsis';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {GroupStatus} from 'sentry/types';
+
+interface Props
+  extends Pick<
+    ReturnType<typeof useListItemCheckboxState>,
+    'countSelected' | 'deselectAll' | 'selectedIds'
+  > {
+  mailbox: ReturnType<typeof decodeMailbox>;
+}
+
+export default function FeedbackListBulkSelection({
+  mailbox,
+  countSelected,
+  selectedIds,
+  deselectAll,
+}: Props) {
+  const {onToggleResovled, onMarkAsRead, onMarkUnread} = useBulkEditFeedbacks({
+    selectedIds,
+    deselectAll,
+  });
+
+  const newMailbox =
+    mailbox === 'resolved' ? GroupStatus.UNRESOLVED : GroupStatus.RESOLVED;
+
+  return (
+    <Flex gap={space(1)} align="center" justify="space-between" style={{flexGrow: 1}}>
```

Comment:
Can we clean up the mixed usage of Flex + inline styles?

Reply:
yeah. `<Flex>` can now accept a `flex={}` prop. 

idk how i feel about this component overall. I do like it better than having to figure out unique names for every combination, but i think Row vs. Col could be built-in choices at least, but idk if that's enough

Reply:
I don't mind that so much. Looking at `feedbackListBulkSelection.tsx` it's all about the html structure -> everything else is stuffed into the hook. 
I think the box model and divs go hand in hand, and flex is all about that (not dealing with colors and fonts and that part of css, also flex properties don't cascade down). 


Coming up with a name for that outer `return <Flex>...</Flex>` is always annoying imo because, like, sometimes i want to call it the same thing as the component itself, or people just call it `<Wrapper>` and move on, imo that's not valuable. I guess i'm thinking that it's not semantic, until we talk about layouts, like `Layout.Header`, or `Layout.Thirds` and those types of examples, which are good. But then the specialization down here... we haven't got a naming convention for it down here.